### PR TITLE
NF: move Preferences.kt and tests to `com.ichi2.anki.preferences` package

### PR DESCRIPTION
**Pull request by BrayanDSO**
## Purpose / Description
Keep things organized in packages

## Approach
- Move the file to `com.ichi2.anki.preferences`
- Fix the imports

## How Has This Been Tested?

CI

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [X] You have commented your code, particularly in hard-to-understand areas
- [X] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by lukstbit — APPROVED** [review]
Ok, it was needed to be done.

File: AnkiDroid/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -261,7 +261,7 @@
             android:configChanges="keyboardHidden|orientation|locale|screenSize|uiMode"
             />
         <activity
-            android:name="com.ichi2.anki.Preferences"
+            android:name="com.ichi2.anki.preferences.Preferences"
```
Comment (david-allison) [inline AnkiDroid/src/main/AndroidManifest.xml:264]: This isn't a breaking change for third parties as it's not exported

**Review by david-allison — APPROVED** [review]
Historically, we had most activities in `.anki`.

I don't oppose this: it's not a hard rule. I'd typically say I'd want to look into how other Android projects handle this, but Android is moving away from activities long-term and don't think this is worth the effort.

**Comment by david-allison** [comment]
I'm leaving this one for @mikehardy's queue as there's potentially a lot of chance for breakages if this isn't merged serially

**Pull request merged**